.github/workflows/deploy-npm.yml

```diff
@@ -157,16 +157,19 @@ jobs:
         if: env.CURRENT_NPM_VERSION != env.NEW_NPM_VERSION && env.NEW_NPM_VERSION != 'not found'
         id: slack
         continue-on-error: true
-        uses: slackapi/slack-github-action@v1.27.0
+        uses: slackapi/slack-github-action@485a9d42d3a73031f12ec201c457e2162c45d02d  # v2.0.0
```

Comment:
This is already whitelisted:
```
slackapi/slack-github-action@* 
```

Therefore, we have the option to keep this action as it is.

Reply:
Yes, but still wanted to make sure that an exact version is for this external action.